src/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
I'm still wondering why this is needed. Doesn't rustbuild put all its stuff into `build/target-triple/...` instead of `src/target`?